pkg/gclient/gclient.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright Kurator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package gclient
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	clusterapiv1beta1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	controlplanev1 "sigs.k8s.io/cluster-api/controlplane/kubeadm/api/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	mcsv1alpha1 "sigs.k8s.io/mcs-api/pkg/apis/v1alpha1"
+
+	"kurator.dev/kurator/pkg/apis/infra/v1alpha1"
+)
+
+// aggregatedScheme aggregates Kubernetes and extended schemes.
+var aggregatedScheme = runtime.NewScheme()
+
+func init() {
+	utilruntime.Must(scheme.AddToScheme(aggregatedScheme))            // add Kubernetes schemes
+	utilruntime.Must(mcsv1alpha1.AddToScheme(aggregatedScheme))       // add mcs-api schemes
+	utilruntime.Must(clusterapiv1beta1.AddToScheme(aggregatedScheme)) // add cluster-api v1beta1 schemes
+	utilruntime.Must(v1alpha1.AddToScheme(aggregatedScheme))          // add v1alpha1-customMachine schemes
+	utilruntime.Must(controlplanev1.AddToScheme(aggregatedScheme))    // add controlplanev1 schemes
```

Comment:
I donot think we should add it again, we have already registered scheme in cmd/cluster-operator/scheme/scheme.go

Reply:
You are right. We dont need to add this file.